lib/std/x/net/TCP.zig

```diff
@@ -0,0 +1,399 @@
+const std = @import("../../std.zig");
+
+const os = std.os;
+const fmt = std.fmt;
+const mem = std.mem;
+const testing = std.testing;
+
+const IPv4 = std.x.os.IPv4;
+const IPv6 = std.x.os.IPv6;
+const Socket = std.x.os.Socket;
+
+/// A generic TCP socket abstraction.
+const TCP = @This();
```

Comment:
Unless I'm blind, there are no toplevel fields in this file and it should therefore be called `tcp.zig` by convention.

Reply:
Gotcha, I'll get that done after CI checks out; waiting to see if any of the unit tests may fail on certain platforms.